first-ruslan/src/test/java/com/ruslan/homework/FactorialTest.java

```diff
@@ -0,0 +1,39 @@
+package com.ruslan.homework;
+
+import com.ruslan.GenerateArrayUtils;
+import com.ruslan.lesson2.homework.factorial.Factorial;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class FactorialTest {
+
+    @Test
+    public void Factorial_must_return_minus_one_if_input_less_than_zero() {
+
+        int n = -2;
+        int input = Factorial.factorial(n);
+        Assertions.assertEquals(-1, input);
+
+    }
+
+    @Test
+    public void Factorial_must_works_correctly() {
+
+        int[] array = GenerateArrayUtils.sortedArray(0, 10);
+
+        int expected_i = 1;
+
+        for (int i = 0; i < array.length; i++) {
+
```

Comment:
new int[]{1,1,2,6,24...}
for (int i = 1; i<ar.len; i++) {
assertEq(ar[i], fact(i))
}